lib/heimdall_tools/cli.rb

```diff
@@ -123,6 +123,17 @@ def netsparker_mapper
       puts options[:output].to_s
     end
 
+    desc 'scoutsuite_mapper', 'scoutsuite_mapper translates Scout Suite results from Javascript to HDF-formatted JSON so as to be viewable on Heimdall'
+    long_desc Help.text(:scoutsuite_mapper)
+    option :javascript, required: true, aliases: '-J'
```

Comment:
I thought about that when I wrote it originally, but left it as uppercase `J` since we're not passing in straight json like every other time, but javascript that we need to muck with before it turns into json.  If you think it's better to be lowercase `j`, I can do that tho.

Reply:
Talked with Eugene and he said that he wanted `-i` and to file an issue to transition the rest as well (or at the very least add an additional alias).

Reply:
Made it `-j` but also added `-i` and `--input` to get ahead of future changes as requested by Eugene

Reply:
See https://github.com/mitre/heimdall_tools/issues/92 and https://github.com/mitre/heimdall_tools/issues/98